modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties

```diff
@@ -117,7 +117,15 @@
     # https://bit.ly/2IkAwwW for Liferay Portal and https://bit.ly/2GIyfZF for
     # Liferay DXP.
     #
-    #liferay.workspace.target.platform.version=7.3.2
+#if (${liferayVersion.startsWith("7.0")})
+    liferay.workspace.target.platform.version=7.0.6
```

Comment:
Does this line need to be commented out too?

Reply:
No, we have to roll back the liferay.workspace.product key for now. Greg is sending a new workspace that will fix that. Until then, we'll default back to using target platform as default. 
But Brian's asking that I send him the change directly